src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
 .container-fullwidth {
  max-width: initial !important;
 }
+
+/*doc
+---
+title: Flex Grids
+name: flexgrid
+parent: grid
+---
+
+```html_example
+<div class="frow mbxl">
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+</div>
+<div class="frow mbxl">
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
```

Comment:
Kinda. That doesn't actually work because of the details of how margins interact with flexbox. That is, not well. So the grid has load-bearing padding, meaning any item needs a div inside it if it's going to have a background color or border.

I'm still trying to figure out how to document this.